### PR TITLE
 Delete URLUtilsReadOnly docs, redirect to WorkerLocation docs 

### PR DESCRIPTION
**Pull request by sideshowbarker**
https://github.com/whatwg/html/commit/32a7a20 completely removed the `URLUtilsReadOnly` interface and moved/inlined/hoisted its members up into the `WorkerLocation` interface. So this change deletes all the `URLUtilsReadOnly` docs, and redirects them to the `WorkerLocation` docs.

Related BCD change: https://github.com/mdn/browser-compat-data/pull/10347

### CODE REVIEW

File: files/en-us/web/api/url_api/index.html [hunk 1]
```diff
@@ -74,7 +74,7 @@ <h2 id="URL_API_interfaces">URL API interfaces</h2>
 </ul>
 </div>
 
-<p>Older versions of the specification included an interface called {{domxref("URLUtilsReadOnly")}}, which has since been merged into the {{domxref("WorkerLocation")}} interface.</p>
+<p>Older versions of the specification included an interface called <code>URLUtilsReadOnly</code>, which has since been merged into the {{domxref("WorkerLocation")}} interface.</p>
```
Comment (wbamberg) [inline files/en-us/web/api/url_api/index.html:77]: Does this sentence still make sense? I'm not really sure what it's adding here.

**Review by wbamberg — COMMENTED** [review]
This looks fine, although I had a question about one sentence that didn't seem very useful any more.

But it looks like all the redirects work, and the redirect entries match the removed files.

File: files/en-us/web/api/url_api/index.html (hunk 1 above)
Comment (sideshowbarker) [reply files/en-us/web/api/url_api/index.html:77]: > Does this sentence still make sense? I'm not really sure what it's adding here.

You’re right, it adds nothing at this point — so I’ve pushed an amended commit that includes dropping that completely.

**Review by chrisdavidmills — APPROVED** [review]
Looks like @sideshowbarker 's latest changes satisfied @wbamberg 's query, so I think I'll go ahead and merge this. Thanks Mike!

**Pull request merged**